warehouse/bigquery/bigquery.go

```diff
@@ -351,6 +430,14 @@ func (bq *HandleT) LoadUserTables() (errorMap map[string]error) {
 		errorMap[warehouseutils.UsersTable] = status.Err()
 		return
 	}
+
+	err = bq.deleteTable(stagingTableName)
```

Comment:
lets move this to defer once its created?

Reply:
also do this on crashRecover. Refer to postgres.go for similar stuff

Reply:
added